utils/colorfiller.py

```diff
@@ -3,66 +3,70 @@
 import os
 import sys
 import re
+import convert
+import importlib
 
-tmpl_ext = '.template'
+USAGE = """Evaluate templates using specified palette.
 
-marker_re = re.compile(r'!!COL(?P<delim>.)(?P<format>.*?)(?P=delim)')
+Takes two arguments - path to a template file or to a folder that will be
+recursively searched for templates and name of the palette module to use."""
```

Comment:
Possibly a bit more flexible design could be to take palette as first argument and then any number of path arguments, or to use option (`--palette`). Well, this is not actually a review comment, its more like a feature suggestion.